src/bootstrap/test.rs

```diff
@@ -925,6 +925,9 @@ impl Step for RustdocGUI {
                     .env("RUSTDOC", builder.rustdoc(self.compiler))
                     .env("RUSTC", builder.rustc(self.compiler))
                     .current_dir(path);
+                if entry.file_name() == "link_to_definition" {
```

Comment:
Could you add a FIXME here that says this?

```rust
// FIXME: implement a `// compile-flags` command or similar
//        instead of hard-coding this test
```